package.json

```diff
@@ -18,12 +18,15 @@
     ],
     "readmeFilename": "README.md",
     "devDependencies": {
-        "grunt": "1.0.1",
-        "fluid-grunt-eslint": "18.1.2"
+        "eslint": "3.12.2",
+        "eslint-config-fluid": "1.0.0",
+        "fluid-grunt-eslint": "18.1.2",
+        "grunt": "1.0.1"
     },
     "dependencies": {
-        "infusion": "2.0.0-dev.20161027T155641Z.0bcd507",
         "flocking": "colinbdclark/flocking#midi-improvements",
```

Comment:
This was branch merged into master in December. Best to use the latest commit hash, 18e94e6fb7a51f0b37c8e69063eb00ff6d1badce (I know, I know). :sheep: